test/plugin/test_out_google_cloud.rb

```diff
@@ -428,7 +428,7 @@ def setup
 
   def create_driver(conf = APPLICATION_DEFAULT_CONFIG, tag = 'test')
     Fluent::Test::BufferedOutputTestDriver.new(
-      Fluent::GoogleCloudOutput, tag).configure(conf, use_v1_config: true)
```

Comment:
This named parameter syntax ought to work. Can we please investigate why it needed to be changed?